src/screens/simulation/PublicSimulation.js

```diff
@@ -293,7 +295,15 @@ const getTimeframesFromValues = ({
     {
         ...defaultParameters,
         r0: initial_r0,
-        start_time: initial_start_date,
+        start_date: subDays(initial_start_date, 15),
```

Comment:
The initial période bar should be much closer to the y axis IMHO

```suggestion
        start_date: subDays(initial_start_date, 5),
```